processor/groupbyattrsprocessor/metrics.go

```diff
@@ -24,9 +24,14 @@ var (
 	mNumGroupedSpans    = stats.Int64("num_grouped_spans", "Number of spans that had attributes grouped", stats.UnitDimensionless)
 	mNumNonGroupedSpans = stats.Int64("num_non_grouped_spans", "Number of spans that did not have attributes grouped", stats.UnitDimensionless)
 	mDistSpanGroups     = stats.Int64("span_groups", "Distributon of groups extracted for spans", stats.UnitDimensionless)
```

Comment:
Would you mind fixing this one as well?

```suggestion
	mDistSpanGroups     = stats.Int64("span_groups", "Distribution of groups extracted for spans", stats.UnitDimensionless)
```